src/model/RecurrentLSTMNetwork.lua

```diff
@@ -12,9 +12,9 @@ return function(opt, params)
    local batchNormalization = opt.batchNormalization or false
    local maxBatchNormalizationLayers = opt.maxBatchNormalizationLayers or 10
 
-   -- container:
+   -- containers:
    params = params or {}
-   layers = layers or {}
```

Comment:
Why do you remove the `layers or {}` line? I think some folks might depend on this (although I do not) -- I believe it lets folks concatenate the output of the recurrent net built in this file to a passed-in set of layers.